test/sqllogictest/transform/aggregation_nullability.slt

```diff
@@ -864,35 +1025,43 @@ WHERE t1.f2 = derived.agg2;
 Explained Query:
   Return // { arity: 1 }
     Project (#1) // { arity: 1 }
-      Join on=(#0 = #2) type=differential // { arity: 3 }
+      Join on=(#0 = #2) type=delta // { arity: 3 }
```

Comment:
This looks like an improvement as we're switching from differential to delta joins?

Reply:
The join had 2 inputs before, and binary joins are always forced to be Differential joins.

Reply:
So actually, maybe a regression, because we went from a tree of two binary joins to one delta join, and that isn't *always* an improvement (delta joins don't have the zig-zag join property of quickly skipping ranges of key non-matches).

Reply:
You mean there is a possibility of this being a regression, but not very likely, right?

This is one example of why we'd need a feature flag for this new transform.

Reply:
No idea if it will be a regression! Any change to MIR is 50-50 it will cause a regression, unless there is a reason it won't. If, for example, we capped the join we introduced here with an `ArrangeBy` I would say this couldn't be a regression. But .. join fusion results in regressions, so could totally be a regression even though locally everything is better.